### ISSUE TITLE
Angular devkit packages not upgrading to latest version

### ISSUE THREAD

**Issue by clundberg1** [angular/angular-cli#14637]
# 🐞 bug report

### Affected Package
@angular-devkit/architect
@angular-devkit/build-webpack

### Description
After following the guide to upgrade the project from Angular 7 to 8, I've run:
`ng update @angular/cli @angular/core `

The problem is that I've noticed a couple of the devkit packages aren't being updated to the latest version. I've tried deleting node_modules and reinstalling everything, but it doesn't seem to work.

## 🌍  Your Environment

Angular CLI: 8.0.1
Node: 12.3.1
OS: win32 x64
Angular: 8.0.0
... animations, common, compiler, compiler-cli, core, forms
... language-service, platform-browser, platform-browser-dynamic
... router

Package Version
@angular-devkit/architect 0.13.8
@angular-devkit/build-angular 0.800.1
@angular-devkit/build-optimizer 0.800.1
@angular-devkit/build-webpack 0.13.8
@angular-devkit/core 7.3.8
@angular-devkit/schematics 8.0.1
@angular/cdk 7.3.7
@angular/cli 8.0.1
@angular/flex-layout 7.0.0-beta.24
@angular/material 7.3.7
@angular/material-moment-adapter 7.3.7
@ngtools/webpack 8.0.1
@schematics/angular 8.0.1
@schematics/update 0.800.1
rxjs 6.5.2
typescript 3.4.5
webpack 4.29.0

</code></pre>

**Anything else relevant?**

**Comment by clydin** [comment]
Please provide the `package.json` for the workspace.

**Comment by clundberg1** [comment]
```
{
    "version": "0.0.0",
    "license": "MIT",
    "scripts": {
        "ng": "ng",
        "start": "ng serve",
        "mock": "ng serve -c=mock",
        "build": "ng build",
        "test": "ng test",
        "lint": "./node_modules/.bin/nx lint && ng lint",
        "e2e": "ng e2e",
        "build:stats": "ng build --stats-json",
        "analyze": "webpack-bundle-analyzer dist/apps/landing/stats.json",
        "start:admin": "ng serve admin -o --port=4201",
        "start:admin:mock": "ng serve admin -o -c=mock --port=4201",
        "build:admin:stats": "ng build admin --stats-json --prod",
        "analyze:admin": "webpack-bundle-analyzer dist/apps/admin/stats.json",
        "start:enterprise": "ng serve enterprise -o --port=4202",
        "start:enterprise:mock": "ng serve enterprise -o -c=mock --port=4202",
        "build:enterprise:stats": "ng build enterprise --stats-json",
        "analyze:enterprise": "webpack-bundle-analyzer dist/apps/enterprise/stats.json",
        "affected:apps": "./node_modules/.bin/nx affected:apps",
        "affected:libs": "./node_modules/.bin/nx affected:libs",
        "affected:build": "./node_modules/.bin/nx affected:build",
        "affected:e2e": "./node_modules/.bin/nx affected:e2e",
        "affected:test": "./node_modules/.bin/nx affected:test",
        "affected:lint": "./node_modules/.bin/nx affected:lint",
        "affected:dep-graph": "./node_modules/.bin/nx affected:dep-graph",
        "affected": "./node_modules/.bin/nx affected",
        "format": "./node_modules/.bin/nx format:write",
        "format:write": "./node_modules/.bin/nx format:write",
        "format:check": "./node_modules/.bin/nx format:check",
        "update": "ng update @nrwl/schematics",
        "update:check": "ng update",
        "workspace-schematic": "./node_modules/.bin/nx workspace-schematic",
        "dep-graph": "./node_modules/.bin/nx dep-graph",
        "help": "./node_modules/.bin/nx help"
    },
    "private": true,
    "dependencies": {
        "@angular/animations": "^8.0.0",
        "@angular/cdk": "^7.3.7",
        "@angular/common": "^8.0.0",
        "@angular/compiler": "^8.0.0",
        "@angular/core": "^8.0.0",
        "@angular/flex-layout": "^7.0.0-beta.24",
        "@angular/forms": "^8.0.0",
        "@angular/material": "^7.3.7",
        "@angular/material-moment-adapter": "^7.3.7",
        "@angular/platform-browser": "^8.0.0",
        "@angular/platform-browser-dynamic": "^8.0.0",
        "@angular/router": "^8.0.0",
        "@mediomelon/ng-core": "0.0.4",
        "@ngez/core": "^5.0.1",
        "@ngrx/effects": "^7.4.0",
        "@ngrx/store": "^7.4.0",
        "@ngrx/store-devtools": "^7.4.0",
        "aws-amplify": "^1.1.25",
        "aws-amplify-angular": "^2.1.12",
        "core-js": "^2.6.3",
        "immer": "^2.1.5",
        "invert-color": "^2.0.0",
        "material-palette": "^1.0.0",
        "moment": "^2.24.0",
        "rxjs": "^6.5.2",
        "web-animations-js": "github:angular/web-animations-js#release_pr208",
        "whatwg-fetch": "^3.0.0",
        "zone.js": "~0.9.1"
    },
    "devDependencies": {
        "@angular-builders/custom-webpack": "^8.0.1",
        "@angular-builders/dev-server": "^7.3.1",
        "@angular-devkit/build-angular": "~0.800.0",
        "@angular/cli": "~8.0.1",
        "@angular/compiler-cli": "^8.0.0",
        "@angular/language-service": "^8.0.0",
        "@nrwl/builders": "7.8.1",
        "@nrwl/schematics": "7.8.1",
        "@types/jasmine": "~2.8.6",
        "@types/jasminewd2": "~2.0.3",
        "@types/jquery": "3.3.6",
        "@types/node": "~8.9.4",
        "codelyzer": "^5.0.1",
        "cypress": "^3.2.0",
        "dotenv": "6.2.0",
        "jasmine-core": "~2.99.1",
        "jasmine-marbles": "0.4.0",
        "jasmine-spec-reporter": "~4.2.1",
        "karma": "~3.0.0",
        "karma-chrome-launcher": "~2.2.0",
        "karma-coverage-istanbul-reporter": "~2.0.1",
        "karma-jasmine": "~1.1.0",
        "karma-jasmine-html-reporter": "^0.2.2",
        "moment-locales-webpack-plugin": "^1.0.7",
        "prettier": "1.16.4",
        "ts-node": "~7.0.0",
        "tslint": "~5.11.0",
        "typescript": "3.4.5",
        "webpack-bundle-analyzer": "^3.3.2"
    }
}
```

**Comment by clydin** [comment]
Neither of the two (`@angular-devkit/architect`, `@angular-devkit/build-webpack`) appear to be present in the `package.json`.  They are most likely present as a result of being dependencies of `@nrwl/builders` which is fixed at `7.8.1` and which requires the following versions:
```
    "@angular-devkit/architect": "~0.13.1",
    "@angular-devkit/build-angular": "~0.13.1",
    "@angular-devkit/build-webpack": "~0.13.1",
```
As a result, the behavior appears correct.

**Comment by clundberg1** [comment]
@clydin 
Are they all supposed to be in the dependencies? Even checking out one of my Angular 7 projects that didn't use Nx, only @angular-devkit/build-angular appeared in the dependencies, so I assumed that's how it's supposed to be.
